packages/shared/src/utils/helpers/__tests__/start-date.ts

```diff
@@ -19,7 +20,8 @@ describe('start_date', () => {
                 start_type: 'spot',
                 submarket: 'major_pairs',
                 underlying_symbol: 'frxAUDJPY',
-                forward_starting_options: [],
+                forward_starting_options:
+                    [] as unknown as ContractsFor['available'][number]['forward_starting_options'],
```

Comment:
```suggestion
                forward_starting_options: [{ blackouts: [], close: '', date: '', open: '' }],
```
Would this work here @kate-deriv?

Reply:
No, the test will fail because `length !== 0` and  array is equal to `[{"sessions": [{"close": "2023-07-26T09:19:03.233Z", "open": "2023-07-26T09:19:03.232Z"}], "text": "Wed - 26 Jul, 2023", "value": NaN}]`